WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -282,7 +282,6 @@ - (void)runStartupSequenceWithLaunchOptions:(NSDictionary *)launchOptions
     [ZendeskUtils setup];
 
     // Networking setup
-    [[AFNetworkActivityIndicatorManager sharedManager] setEnabled:YES];
```

Comment:
Are we using the Alamofire `NetworkActivityIndicatorManager`? Or just not showing a network activity indicator? 

Just curious about this change from a UX perspective :)

Reply:
So we were previously using the AFNetworking indicator but when moved our API classes ( more than an year ago) the indicator was not being presented in a lot of places. This PR now completely remove the AFNetworking indicator manager.

As @frosty say I can do a follow up PR where I add the indicator again using the AlamoFire submodule specific for this work.

The indicator, is a iOS system thing, it is just the small spinning indicator that shows up on the  top left of the status bar when network calls are being made.

Reply:
Ok so I had a go on adding the network indicator using [AlamoFire sub spec](https://github.com/Alamofire/AlamofireNetworkActivityIndicator) and it was pretty easy.

The only thing that kept me down was the fact that iOS doesn't show the network indicator on the iPhone X XS and XR... so for an hour I thought it was not working... After I switch to an iPhone 6 all was good :)